templates/stream_processor.yaml

```diff
@@ -44,6 +45,10 @@ Parameters:
   JobTableStreamArn:
     Description: Stream ARN for Jobs Table
     Type: String
+  LambdaJobsMemorySize:
+    Type: Number
+    MinValue: 128
+    MaxValue: 10240
```

Comment:
Nitpick: I don't think we need Min/Max validation here since it's done in the parent stack.

Reply:
making cfn-lint happy